cmd/albatross/albatross.go

```diff
@@ -6,10 +6,11 @@ import (
 
 	"github.com/gorilla/mux"
 
+	"github.com/gojekfarm/albatross/pkg/api"
```

Comment:
we'll end up changing this to `github.com/gojek` again later.

Reply:
Ack